package.json

```diff
@@ -52,14 +53,18 @@
     "type": "git",
     "url": "git+https://github.com/bwinton/SnoozeTabs.git"
   },
+  "config": {
+    "GA_TRACKING_ID": "UA-90911916-1"
+  },
   "scripts": {
     "start": "npm run build && npm-run-all --parallel test watch run storybook",
     "start-win": "npm run build && npm-run-all --parallel test watch",
-    "build": "npm run clean && npm-run-all --parallel copy:assets bundle:*",
+    "build": "npm run clean && npm-run-all --parallel copy:* bundle:*",
     "run": "web-ext run -s dist --firefox=nightly",
     "clean": "rm -rf dist && shx mkdir -p dist/popup",
     "watch": "npm-run-all --parallel watch:*",
     "copy:assets": "shx cp -r src/manifest.json src/icons dist/ && svgo src/icons -o dist/icons && shx cp -r src/popup/*.html src/popup/FiraSans-Regular.* dist/popup/",
+    "copy:js": "shx cp node_modules/testpilot-metrics/testpilot-metrics.js dist/",
```

Comment:
Not super happy with copying this over as a script, rather than importing it as a module. But, testpilot-metrics does some conditional `require()` stuff to work for WebExtensions, SDK add-ons, and Test Pilot itself - and rollup doesn't like that.